app/models/manageiq/providers/openstack/infra_manager/orchestration_stack.rb

```diff
@@ -76,4 +76,67 @@ def post_scaledown_task(services, task_id = nil)
     raise MiqException::MiqQueueRetryLater.new(:deliver_on => Time.now.utc + 1.minute) unless raw_status.first == 'UPDATE_COMPLETE'
     services.each(&:delete_service)
   end
+
```

Comment:
Changes in this files don't look to be related to Flavor operations.

Reply:
fixed